manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
```

```suggestion
PackageIdentifier: athlabs.spyglass
```

Also modify the file name and folder name to match it.